### PR TITLE
Add missing security context statement

### PR DESCRIPTION
**Pull request by sousadax12**
Add Security Context statement to wait-for-db and change-password

### CODE REVIEW

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]
Hi @sousadax12 ,

thanks for your contribution and making a effort in passing the QA. Your code looks good so far, but please mention your changes in the `Chart.yaml` so it is displayed on artifacthub and in `CHANGELOG.md`

File: charts/sonarqube/templates/change-admin-password-hook.yml [hunk 1]
```diff
@@ -43,6 +43,10 @@ spec:
       containers:
       - name: {{ template "sonarqube.fullname" . }}-change-default-admin-password
         image: {{ default "curlimages/curl:latest" .Values.curlContainerImage }}
+        {{- if $securityContext := .Values.initContainers.securityContext }}
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/templates/change-admin-password-hook.yml:46]: i missed that initially but here again. reusing the `initContainers` scope is misleading

File: charts/sonarqube/templates/deployment.yaml [hunk 2]
```diff
@@ -174,6 +174,10 @@ spec:
         - name: "wait-for-db"
           image: {{ default "busybox:1.32" .Values.initContainers.image }}
           imagePullPolicy: {{ .Values.image.pullPolicy  }}
+          {{- if $securityContext := .Values.initContainers.securityContext }}
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/templates/deployment.yaml:177]: reusing the `initContainers` scope is misleading

File: charts/sonarqube/templates/sonarqube-sts.yaml [hunk 3]
```diff
@@ -64,6 +64,10 @@ spec:
         - name: "wait-for-db"
           image: {{ default "busybox:1.32" .Values.initContainers.image }}
           imagePullPolicy: {{ .Values.image.pullPolicy  }}
+          {{- if $securityContext := .Values.initContainers.securityContext }}
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/templates/sonarqube-sts.yaml:67]: reusing the `initContainers` scope is misleading

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]

File: charts/sonarqube/templates/deployment.yaml (hunk 2 above)
Comment (sousadax12) [reply charts/sonarqube/templates/deployment.yaml:177]: Hi! Isn't this a initContainer? Should I create a value like wait-for-db.securityContext ?

File: charts/sonarqube/templates/sonarqube-sts.yaml (hunk 3 above)
Comment (sousadax12) [reply charts/sonarqube/templates/sonarqube-sts.yaml:67]: Hi! Isn't this a initContainer? Should I create a value like wait-for-db.securityContext ?

File: charts/sonarqube/templates/change-admin-password-hook.yml (hunk 1 above)
Comment (sousadax12) [reply charts/sonarqube/templates/change-admin-password-hook.yml]: Created a account.securityContext value

File: charts/sonarqube/templates/sonarqube-sts.yaml (hunk 3 above)
Comment (tobias-trabelsi-sonarsource) [reply charts/sonarqube/templates/sonarqube-sts.yaml:67]: ah yeah you are right. sorry too many topics currently

File: charts/sonarqube/values.yaml [hunk 4]
```diff
@@ -488,6 +488,7 @@ extraConfig:
 # account:
 #   adminPassword: admin
 #   currentAdminPassword: admin
+#   securityContext: {}
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/values.yaml:491]: this needs to be documented in the readme of the chart as well

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]

**Review by tobias-trabelsi-sonarsource — APPROVED** [review]
LGTM. Thanks for the contribution

**Pull request merged**